src/http/http_endpoint.h

```diff
@@ -320,7 +320,7 @@ namespace http
         }
 
         // On any exception, close the connection.
-        LOG_TRACE_FMT("Closing connection due to exception: {}", e.what());
+        LOG_FAIL_FMT("Closing connection due to exception: {}", e.what());
```

Comment:
This is good for debugging, but probably not something we want in the final build because it may impact confidentiality.

Reply:
Done now.